zenstruck/messenger-monitor-bundle/0.5/src/Controller/Admin/MessengerMonitorController.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Controller\Admin;
```

Comment:
Please consider making it strict to adhere modern coding standards.

```suggestion
<?php

declare(strict_types=1);

namespace App\Controller\Admin;
```

Reply:
I don't personally use strict types: not possible to please everyone here, unfortunately